vector/src/main/java/im/vector/app/features/spaces/manage/SpaceManagedSharedAction.kt

```diff
@@ -25,4 +25,5 @@ sealed class SpaceManagedSharedAction : VectorViewModelAction {
     object CreateRoom : SpaceManagedSharedAction()
     object ManageRooms : SpaceManagedSharedAction()
     object OpenSpaceAliasesSettings : SpaceManagedSharedAction()
+    object OpenRoomPermissionSettings : SpaceManagedSharedAction()
```

Comment:
Maybe you have hesitated with `OpenSpacePermissionSettings`?

Reply:
Done